docker/pravega/Dockerfile

```diff
@@ -13,27 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM openjdk:11.0.8-jre-slim
+FROM adoptopenjdk/openjdk11:jre-11.0.11_9-alpine
 
-RUN apt-get update && apt-get install -y -q \
```

Comment:
why are you removing this useful tools?

Reply:
I don't see us using requiring them (yet to be proved by testing) hence no reason to keep them in the image. The lesser components installed the smaller final docker image and lesser chances of getting CVEs found

Reply:
makes sense to me.

we are running the pods as root, correct ? so the user is able to add debugging tools easily
 
if we switch to root-less docker images then we will have to provide debugging tools out of the box probably, otherwise it will be very difficult to do troubleshooting of problems

Reply:
wouldn't be two different cases: running a pod as root and be logged in as root inside a pod?

Reply:
you cannot login as root in a rootless container, so you cannot install additional tools (there are some tricks but they are hard and not for the regular sys admins)

btw we are not still targeting rootless containers so it is fine to have the smallest image as possible